src/Stratis.SmartContracts.Standards/IStandardToken.cs

```diff
@@ -8,22 +8,22 @@ public interface IStandardToken
         /// <summary>
         /// The total number of tokens in existence.
         /// </summary>
-        uint TotalSupply { get; }
+        ulong TotalSupply { get; }
 
         /// <summary>
         /// Gets the balance of the specified address.
         /// </summary>
         /// <param name="address">The address to query the balance of.</param>
         /// <returns>Balance for the given address</returns>
-        uint GetBalance(Address address);
+        ulong GetBalance(Address address);
 
         /// <summary>
         /// Transfers tokens owned by the sender to the destination address.
         /// </summary>
         /// <param name="to">The address to transfer the tokens to.</param>
         /// <param name="amount">The amount of tokens to transfer.</param>
         /// <returns>A bool representing whether the transfer was successful.</returns>
-        bool Transfer(Address to, uint amount);
+        bool TransferTo(Address to, ulong amount);
```

Comment:
What's the idea behind this?

Reply:
Prevent confusion with the `Transfer` method that exists on all contracts.